### PR TITLE
home feature feed

### PR DESCRIPTION
**Pull request by vinnyjth**
**IMPORTANT: If this is for testing code in our npm dependencies, you should be pointing your PR at the `canary` or `next` branch. `master` should never rely on unreleased code.**

### CODE REVIEW

File: apolloschurchapp/src/tabs/home/Home.js [hunk 1]
```diff
@@ -29,43 +52,25 @@ class Home extends PureComponent {
...
+                    openUrl={openUrl}
+                    navigation={this.props.navigation}
+                    nodeId={data?.homeFeedFeatures?.id}
+                    onPressActionItem={handleOnPress}
+                    ListHeaderComponent={
+                      <LogoTitle source={require('./wordmark.png')} />
+                    }
+                  />
+                )}
+              </Query>
```
Comment (redreceipt) [inline apolloschurchapp/src/tabs/home/Home.js:73]: this makes me so happy

Comment (vinnyjth) [reply apolloschurchapp/src/tabs/home/Home.js:73]: 😂  me too

**Comment by redreceipt** [comment]
looks good, blocked until the core PR makes it into canary

**Review by redreceipt — APPROVED** [review]

**Comment by vinnyjth** [comment]
Looks like the canary build made it out.

**Comment by vinnyjth** [comment]
@redreceipt What's the correct step now? Swap the package.json to point at the canary packages?

**Comment by redreceipt** [comment]
yep, we can run `yarn canary` to update packages and test pointed to the live server

**Comment by redreceipt** [comment]
not sure why the "check api" workflow isn't running, that should tell us it's ready

**Pull request merged**